README.md

```diff
@@ -34,36 +35,160 @@ npm install --save @webex/components @webex/sdk-component-adapter
 
 ## Usage
 
+### Setup
+
+When using the Webex SDK Adapter, it is expected to have a fully authenticated SDK instance passed to it:
+
+```js
+const webex = new Webex({
+  credentials: `<YOUR WEBEX ACCESS_TOKEN>`,
+});
+
+const adapter = new WebexSDKAdapter(webex);
+```
+
+#### Connect
+
+The Webex SDK requires different connections to be setup in order for it to be usable.
+Some examples of these connections include device registration and web socket connections.
+
+These connections are handled by the SDK Adapter, but require the `connect` function to be manually called.
+
+```js
+const webex = new Webex({
+  credentials: `<YOUR WEBEX ACCESS_TOKEN>`,
+});
+
+const adapter = new WebexSDKAdapter(webex);
+
+adapter.connect();
+```
+
+#### Disconnect
+
+When the adapter is no longer going to be used, the `disconnect` function will do the necessary tear-down of all the connections created in the `connect` function:
+
+```js
+adapter.disconnect();
+```
+
 ### Components
 
+The Webex SDK Adapter is meant to be used with Webex Components.
+
 For more information on how to use **Webex Components**, visit [this page](https://github.com/webex/components/blob/master/README.md).
 
-Putting everything together - SDK adapters and components - this is a simple example of how using a component would look like:
+The following examples display how you can utilize the Webex SDK Adapter along with Webex Components to provide a fully connected component:
+
+#### React Component With Hooks
+
+Utilizing the `useEffect` hook, we can connect our adapter in a deferred event after initial render.
 
 ```js
 import '@webex/components/dist/webexComponents.css';
 
-import React from 'react';
-import ReactDOM from 'react-dom';
+import React, {useEffect, useState} from 'react';
 import Webex from 'webex';
 import WebexSDKAdapter from '@webex/sdk-component-adapter';
-import {WebexAvatar} from '@webex/components';
+import {WebexAvatar, WebexDataProvider} from '@webex/components';
 
 const webex = new Webex({
-  credentials: <YOUR WEBEX ACCESS_TOKEN>,
+  credentials: `<YOUR_ACCESS_TOKEN>`,
 });
 
-const adapter = new WebexSDKAdapter(webex);
+function App() {
+  const [isReady, setReady] = useState(false);
+
+  useEffect(() => {
+    // This is the suggested way to do async hooks.
+    // Ref: https://github.com/facebook/react/issues/14326
+    async function doConnect() {
+      const adapter = new WebexSDKAdapter(webex);
```

Comment:
You need to move the declaration of `adapter` out of this function so the cleanup call below can use it.